scripts/ubiquity_lead_scorer.py

```
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+"""GitHub Bounty Lead Scorer — ranks by engagement delta, not keyword volume."""
+import os, sys, json, csv, re, argparse
+from datetime import datetime, timedelta
+from urllib.request import Request, urlopen
+from urllib.parse import urlencode
+
+GITHUB_API = "https://api.github.com"
+TOKEN = os.environ.get("GITHUB_TOKEN", "")
+HEADERS = {"Accept": "application/vnd.github+json", "User-Agent": "ubiquity-lead-scorer/1.0"}
+if TOKEN: HEADERS["Authorization"] = f"Bearer {TOKEN}"
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)